src/SFA.DAS.AssessorService.Web/Views/ApplyForWithdrawal/WithdrawalApplications.cshtml

```diff
@@ -25,11 +25,10 @@
                         <th class="govuk-table__header" scope="col">Type</th>
                         <th class="govuk-table__header" scope="col">Name</th>
                         <th class="govuk-table__header" scope="col">Status</th>
-                        <th class="govuk-table__header govuk-table__header--numeric" scope="col">Action</th>
+                        <th class="govuk-table__header" scope="col"></th>
```

Comment:
Is this supposed to be a blank header? The screenshots on the ticket do show the Action header

Reply:
Good spot - I synced this with the other screen of the same type which does not show the word 'Action' - Assume this is ok but will go over it will Dil and Raj

Reply:
This was incorrectly removed - restored the 'Action' header and right aligned both Status and Action to match similar page on apply journey